pages/common/cheatshh.md

```diff
@@ -0,0 +1,33 @@
+# cheatshh
+
+> CLI cheatsheet with customized descriptions, tldr and groups, to look into for your reference.
+> Press Enter to a command to copy it to your clipboard and exit.
+> More information: <https://github.com/AnirudhG07/cheatshh>.
+
+- Add a new command independently or in a group:
+
+`cheatshh -a`
+
+- Edit an existing command description & possibly to add it in a group:
+
+`cheatshh -ec`
+
+- Delete a command:
+
+`cheatshh -dc`
+
+- Create a new group:
+
+`cheatshh -g`
+
+- Edit a group's name or description:
+
+`cheatshh -eg`
+
+- Delete a group:
+
+`cheatshh -dg`
+
+- See the man pages as well:
+
+`cheatshh -m`
```

Comment:
I would avoid putting these changes, since the new people in coding might understand the meaning of [a]dd, and for them -a is much easier.<br>
Although I will make minor changes in the texts as suggested. <br>
@sebastiaanspeck do you suggest me to put both?  Eg- <br>
```
- TEXT
`cheatshh -a `   # OR  `cheatshh --add`
```
Although this might be against your style. <br>
Also thanks for the man page correction!